libnetwork/drivers_unsupported.go

```diff
@@ -2,6 +2,6 @@
 
 package libnetwork
 
-func getInitializers() []initializer {
+func registerNetworkDrivers(r driverapi.Registerer, driverConfig func(string) map[string]interface{}) error {
```

Comment:
Not a blocker; I guess we should start using `any` in place of `interface{}` now (still getting used to that myself).